src/tools/pkcs11-tool.c

```diff
@@ -5043,8 +5046,8 @@ static int test_signature(CK_SESSION_HANDLE sess)
 	}
 
 	if (firstMechType == CKM_RSA_X_509) {
-		/* make sure our data is smaller than the modulus */
-		data[0] = 0x00;
+		/* make sure our data is smaller than the modulus - 11 */
+		memset(data, 0, 11); /* in effect is zero padding */ 
```

Comment:
clang-format suggested the following change:


```
-		memset(data, 0, 11); /* in effect is zero padding */ 
+		memset(data, 0, 11); /* in effect is zero padding */
```

<!-- identifier: clang-format -->
<!-- File: src/tools/pkcs11-tool.c -->